src/main/java/com/ibm/watson/developer_cloud/visual_recognition/v3/VisualRecognition.java

```diff
@@ -92,16 +92,15 @@ public VisualRecognition(String versionDate) {
   }
 
   /**
-   * Instantiates a new Visual Recognition V3 service by username and password.
+   * Instantiates a new Visual Recognition V3 service with a given API key.
    *
    * @param versionDate The version date (yyyy-MM-dd) of the REST API to use. Specifying this value
    *        will keep your API calls from failing when the service introduces breaking changes.
-   * @param username the username
-   * @param password the password
+   * @param apiKey the api key
    */
-  public VisualRecognition(String versionDate, String username, String password) {
+  public VisualRecognition(String versionDate, String apiKey) {
```

Comment:
New unit tests for the constructors found that this constructor is **always** throwing an Exception. In the spirit of *fail-fast programming* I introduced breaking API changes to remove the obsolete constructor.